src/lib/flow/actions.js

```diff
@@ -69,14 +69,14 @@ const configureFCL = (wallet) => {
       .put("discovery.wallet", import.meta.env.VITE_DAPPER_DISCOVERY)
       .put("discovery.wallet.method", "POP/RPC")
   }
-  else if (wallet === 'other') {
+  else {
     config()
       .put("discovery.wallet", import.meta.env.VITE_FCL_DISCOVERY)
       .put("discovery.wallet.method", "IFRAME/RPC")
   }
 }
 
-export const configureFCLAndLogin = async (wallet) => {
+export const configureFCLAndLogin = async (wallet, service) => {
   currentWallet.set(wallet);
   configureFCL(wallet);
   await fcl.authenticate();
```

Comment:
I think this line is missing something, it throws an error. 

<img width="418" alt="image" src="https://user-images.githubusercontent.com/27052451/192463781-1a377e0b-d870-4d4a-81bb-78d2f6eda5fa.png">

I assume it needs to be changed to: `fcl.authenticate({service: service})`. that makes it it works for Lilico. 

However Blocto still doesn't work, for another reason (throws 404). Any ideas?

<img width="412" alt="image" src="https://user-images.githubusercontent.com/27052451/192463979-97667c46-bcd0-4f76-879e-0ce15adef838.png">